codegen/build/build.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2025 Heroic Labs.
+// All rights reserved.
+//
+// NOTICE: All information contained herein is, and remains the property of Heroic
+// Labs. and its suppliers, if any. The intellectual and technical concepts
+// contained herein are proprietary to Heroic Labs. and its suppliers and may be
+// covered by U.S. and Foreign Patents, patents in process, and are protected by
+// trade secret or copyright law. Dissemination of this information or reproduction
+// of this material is strictly forbidden unless prior written permission is
+// obtained from Heroic Labs.
+
+package api
+
+//build:ignore
```

Comment:
I think we're missing this:
```
//go:generate go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2@30552a56c2c4caa0ae7cb769abe50b1f0cf0ac75
```
to make sure this uses the right version of the swagger tooling that is compatible with the code generator.

Reply:
ah but we do need that for Nakama